src/store/profile/index.ts

```diff
@@ -18,6 +19,8 @@ import { IState } from './types';
 
 const initialState: IState = {
 	loading: true,
+	loadingRequisites: true,
+	loadingTemplates: true,
```

Comment:
Изначальный state для loading должен быть false и уже при pending ставится true, а когда успешный запрос в fulfilled или провальный в rejected - должно ставится false
```suggestion
	loadingRequisites: false,
	loadingTemplates: false,
```

Reply:
Если оставить true и что-то пойдет не так, то у тебя будет висеть бесконечный лоадер